src/storage/sdr/record/mod.rs

```diff
@@ -64,7 +64,9 @@ impl SensorKey {
         let owner_channel_fru_lun = record_data[1];
 
         // NOTE(unwrap): value is guaranteed to be in the correct range due to mask + shift.
```

Comment:
Please remove this (outdated & incorrect) comment

Reply:
Done.